lonelyplanet/bane
# re-position the notification bubble

Navigation refactoring changed the notification widget positioning ... :D

This PR attempt to fix it...

![screen shot 2015-04-27 at 4 29 02 pm](https://cloud.githubusercontent.com/assets/2614324/7358438/e592864c-ecfa-11e4-9473-db42b32eeb80.png)

![screen shot 2015-04-27 at 4 29 10 pm]
(https://cloud.githubusercontent.com/assets/2614324/7358446/ec03d94a-ecfa-11e4-80ba-3df1d08eeb3e.png)


 To verify:

goto shop.lonelyplanet.com add stuff to your cart
add hosts file entry `127.0.0.1 lonelyplanet.com` so shop cookie can be read
visit page lonelyplanet.com:3000/asia
verify that notification bubble is positioned correctly (mobile/table/desktop view)
